indico/modules/users/templates/personal_data.html

```diff
@@ -15,11 +15,63 @@
                                                user-last-name="{{ user.last_name }}">
                         </ind-user-block-button>
                         {% if allow_deletion %}
-                            <ind-user-delete-button user-id="{{ (user.id if user != session.user else none) | tojson }}"
-                                                    user-is-admin="{{ user.is_admin | tojson }}"
-                                                    user-first-name="{{ user.first_name }}"
-                                                    user-last-name="{{ user.last_name }}">
-                            </ind-user-delete-button>
+                            {% set is_same_user = user == session.user %}
+                            {% if user.is_admin and not is_same_user %}
+                                {% set disabled_message = _('You cannot delete an admin account') %}
+                            {% elif is_same_user %}
+                                {% set disabled_message = _('You cannot delete your own account') %}
+                            {% else %}
+                                {% set dialog_title = _('Delete') + ' {first_name} {last_name}?'.format(first_name=user.first_name, last_name=user.last_name) %}
+                            {% endif %}
+                            <ind-delete-button dialog-title="{{ dialog_title }}"
+                                               button-label="{% trans %}Delete User{% endtrans %}"
+                                               is-disabled="{{ user.is_admin | tojson }}"
+                                               disabled-message="{{ disabled_message }}"
+                                               message="{% trans %}Deleted user accounts cannot be restored.{% endtrans %}"
+                                               cancel-text="{% trans %}No, I don't{% endtrans %}"
+                                               confirm-text="{% trans %}Yes, I want to delete{% endtrans %} {{ user.first_name }}?"
+                                               record-name="{{ user.first_name }} {{ user.last_name }}"
+                                               delete-url="{{ url_for('users.user_delete', user) }}"
+                                               redirect-url="{{ url_for('users.users_admin') }}">
+                                {% trans %}Once deleted, the following will happen:{% endtrans %}
+                                <div style="padding-top: 15px; padding-bottom: 15px">
+                                    <ul class="ui list" style="list-style: none; padding-left: 0; margin-top: 0">
```

Comment:
I don't think it's a common practice in the project to add in-line style for HTML in Jinja templates? You may want to check if there are classes that apply standardized paddings/margins, etc.

Reply:
Hi @ThiefMaster, I didn't find any class that already has these styles, shall I create one or let it be inline styles?

Reply:
ping @ThiefMaster

Reply:
ping @ThiefMaster

Reply:
if there's a decent place feel free to add one